server/app/models/ApplicationStatusesModel.java

```diff
@@ -0,0 +1,55 @@
+package models;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.ebean.annotation.DbJson;
+import io.ebean.annotation.WhenCreated;
+import java.time.Instant;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import play.data.validation.Constraints;
+import services.program.StatusDefinitions;
+
+/** An EBean class that stores all the statuses related to programs. */
+@Entity
+@Table(name = "application_statuses")
+public class ApplicationStatusesModel extends BaseModel {
```

Comment:
**suggestion**: Re my comment, naming this `ApplicationStatusSetModel` and the table `application_status_sets` helps clean up the naming pattern.

Reply:
nvm, per my comment on the main PR